src/components/view/DocumentDetailView/DocumentDetailView.tsx

```
@@ -0,0 +1,157 @@
+
+
+  useInitializeView(() => {
+    initialize();
+  }, [], []);
+
+  return (
+    <Layout colorScheme={props.colorScheme} >
+      {(!documentDetail) && <LoadingIndicator />}
+      {documentDetail?.showDownload &&
+        <div className="mdhui-survey-answer-file-download-button">
+          <Button onClick={() => downloadFile()} fullWidth={false} children={language("download")} ></Button>
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)